src/components/Panel.ts

```diff
@@ -275,10 +286,19 @@ class Panel implements FlickingPanel {
     return this.state.size;
   }
 
-  public getBbox(): ClientRect {
+  public getBbox(): BoundingBox {
     const state = this.state;
     if (!state.cachedBbox) {
-      state.cachedBbox = this.element.getBoundingClientRect();
+      if (!this.element.parentNode) {
+        this.viewport.getCameraElement().appendChild(this.element);
```

Comment:
@WoodNeck Isn't there possibility to be called although `renderExternal = true`?

Reply:
You're right, that can happen when Flicking's resize is called on frameworks.
I'll fix it.